app/domain/logs.rb

```diff
@@ -222,6 +222,11 @@ module Util
         code: "CONJ00023D"
       )
 
+      PrintConjurVersionStartingUp = ::Util::TrackableLogMessageClass.new(
```

Comment:
this can change to `ConjurVersionStartingUp` or `ConjurVersionStartup`. The `print` is implicit.